doc/user-manual/language/mixfix-operators.lagda.rst

```diff
@@ -17,23 +17,43 @@
 Mixfix Operators
 ****************
 
-A name containing one or more name parts and one or more ``_`` can be used as an operator where the arguments go in place of the ``_``. For instance, an application of the name ``if_then_else_`` to arguments ``x``, ``y``, and ``z`` can be written either as a normal application ``if_then_else_ x y z`` or as an operator application ``if x then y else z``.
-
-Examples:
+A type name, function name, or constructor name can comprise one or more name
+parts if we separate them with one or more underscore characters ``_``, and the
+resulting name can be used as an operator. From left to right, each argument
+goes in the place of each underscore ``_``.
+
+For instance, we can join with underscores the name parts ``if``, ``then``,
+and ``else`` into a single name ``if_then_else_``. The application of the
+function name ``if_then_else_`` to some arguments named ``x``, ``y``, and ``z``
+can still be written either as:
+
+* a standard application by using the full name ``if_then_else_ x y z``
+* an operator application by placing the arguments between the name parts
+  ``if x then y else z``, leaving a space between arguments and part names
+
```

Comment:
While you are at it, you could include _sections_. All alternatives are:
```agda
if x then y else z
(if_then y else z) x
(if x then_else z) y
if x then y else_ z
if x then_else_ y z
if_then y else_ x z
(if_then_else z) x y
if_then_else_ x y z
```
Please add some of the section alternatives (leaving some underscores).